templates/default.mustache

```diff
@@ -78,13 +78,13 @@ put_template 21 $color21
 # foreground / background / cursor color
 if [ -n "$ITERM_SESSION_ID" ]; then
   # iTerm2 proprietary escape codes
-  put_template_custom Pg {{base05-hex}} # foreground
-  put_template_custom Ph {{base00-hex}} # background
-  put_template_custom Pi {{base05-hex}} # bold color
-  put_template_custom Pj {{base02-hex}} # selection color
-  put_template_custom Pk {{base05-hex}} # selected text color
-  put_template_custom Pl {{base05-hex}} # cursor
-  put_template_custom Pm {{base00-hex}} # cursor text
+  put_template_custom Pg $(sed -e 's#/##g' <<<$color07) # foreground
```

Comment:
This script uses the `/bin/sh` shebang, so it shouldn't use a bashism like `<<<`. It should consist solely of POSIX-compliant shell to keep compatibility with the most shells. Ex. dash is a POSIX-only shell.